src/Elmish.WPF/Binding.fs

```diff
@@ -232,6 +232,21 @@ module internal Helpers =
     { Name = binding.Name
       Data = BindingData.box binding.Data }
 
+  let createOneWaySeqLazyData
+    (get: 'model -> 'a)
+    (equals: 'a -> 'a -> bool)
+    (map: 'a -> #seq<'b>)
+    (itemEquals: 'b -> 'b -> bool)
+    (getId: 'b -> 'id) =
+      let x = OneWaySeqLazyData {
```

Comment:
Style-wise, I normally follow the rule that the function body should be indented one level (2 spaces), and the parameters, if appearing on separate lines such as this, should be indented three levels (one level further than the function name):

```f#
let createOneWaySeqLazyData
      (get: 'model -> 'a)
      (equals: 'a -> 'a -> bool)
      (map: 'a -> #seq<'b>)
      (itemEquals: 'b -> 'b -> bool)
      (getId: 'b -> 'id) =
  let x = OneWaySeqLazyData {
```

Reply:
Ok.  I will keep that in mind for future reference.  It doesn't matter anymore here because I replace the define-an-addition-function workaround with the reorder-the-methods workaround.